src/periph/rv_dm/dv/sva/rv_dm_sva.core

```diff
@@ -21,9 +21,9 @@ targets:
     filesets:
       - files_dv
 
-  formal:
-    <<: *default_target
-    filesets:
-      - files_formal
-      - files_dv
-    toplevel: rv_dm
+  #formal:
```

Comment:
Any reason to do this?

Reply:
We have seen the warning in the initial fusesoc simulation about "formal" section in targets of ibex core files. So I've commented out all of them, Not sure to del something from the pre-build core files. should I delete these lines?